articles/user-profile/index.md

```diff
@@ -4,114 +4,116 @@ url: /user-profile
 
-The Raw JSON page displays all of the information contained on the user's profile in JSON format. This is an easy way to quickly view all of the information available about the user.
+The Raw JSON tab displays all of the information contained on the user's profile in JSON format so you can quickly view all of the available information about the user.
 
-### Impersonating a User
+## Impersonating a User
 
-Sometimes you need to impersonate a user; login to your app as the user, see everything exactly as the user sees it, and do everything exactly as the user does it. You can do this using the Management Dashboard. 
+If you need to log in to your app as a user, see everything exactly as the user sees it, and do everything exactly as the user does it, you can do this using the Dashboard. 
 
-Navigate to the [Users](${uiURL}/#/users) page in the Management Dashboard and select the user you want to login as. Click on the _Sign in as User_ and select the app you want to log into using the dropdown menu.
+Navigate to the [Users](${uiURL}/#/users) page in the Dashboard. Select the user you want to impersonate from the list. When the User Details page opens, click the "Sign in as User" button. Select the app you want to log into from the dropdown menu.
 
 ![](/media/articles/user-profile/signin-as-user-01.png)
 
-> Can't see the button? Check the following conditions; they should apply for the button to be displayed:
+> Can't see the button? The following conditions are required for the button display:
 > - The applications registered in the account must have at least one callback URL listed.
-> - The applications must have the connections turned on that the users who are to be impersonated belong to.
+> - The applications must have the connections that the impersonated user belongs to turned on.
 
-A popup displays the URL to be used in order to impersonate the user. You can choose either to copy the URL into the clipboard (white button) or open it in a separate browser tab/window (blue button).
+A popup will display the URL to use to impersonate the user. You can copy the URL into the clipboard (white button) or open the URL in a separate browser tab/window (blue button).
 
 ![](/media/articles/user-profile/signin-as-user-02.png)
 
-> You can also use the [Impersonation API](/api/authentication#!#post--users--user_id--impersonate). The API generates a link that can be used once to log in as a specific user. To distinguish between real logins and impersonation logins, the profile of the impersonated user will contain additional `impersonated` and `impersonator` properties.
+> You can also use the [Impersonation API](/api/authentication#!#post--users--user_id--impersonate) to impersonate a user. The API generates a link that you can used once to log in as a specific user. To distinguish between real logins and impersonation logins, the profile of the impersonated user will contain additional `impersonated` and `impersonator` properties.
```

Comment:
This and the following paragraph might be better in a panel.
::: panel-info Impersonation API
You can also use the [Impersonation API](/api/authentication#!#post--users--user_id--impersonate) to impersonate a user. The API generates a link that you can used once to log in as a specific user. To distinguish between real logins and impersonation logins, the profile of the impersonated user will contain additional `impersonated` and `impersonator` properties. Please see [User Impersonation](/user-profile/user-impersonation) for more information about impersonating a user.
:::